ReactWindows/ReactNative/ReactApplicationDelegate.cs

```diff
@@ -43,6 +44,14 @@ public class ReactApplicationDelegate
             _application.EnteredBackground += OnEnteredBackground;
         }
 
+        /// <summary>
+        /// The initial props for the React app.
+        /// </summary>
+        public virtual JObject LaunchOptions
+        {
+            get;
+        }
```

Comment:
Other than to override `LaunchOptions`, are apps expected to subclass this class? If not, is there something that apps already subclass where this hook would fit in?

Reply:
@rigdern the reason for the `ReactApplicationDelegate` subclass is so that apps can more easily embed react-native-windows without necessarily needing to use `ReactApplication`. E.g., the app may already be using some framework that requires a custom `Application` class.